vcd/resource_vcd_network_routed_v2_nsxt_test.go

```diff
@@ -170,6 +175,7 @@ resource "vcd_network_routed_v2" "net1" {
 
   gateway = "1.1.1.1"
   prefix_length = 24
+  guest_vlan_allowed = true
```

Comment:
For consistency we keep these configuration templates in line with Terraform style guidelines
```suggestion
  gateway                    = "1.1.1.1"
  prefix_length            = 24
  guest_vlan_allowed = true
```